### PR TITLE
Revised "winning over engineering leaders" Docs page

### PR DESCRIPTION
**Pull request by caitlinthefirst**
## Description

I've drafted a "Winning Over Engineering Leaders" stakeholder page to replace the current stub. This follows the format present on the "Winning Over Marketers" and "... Developers" pages. Please let me know if I've missed the mark or need any edits. 

## Related Issues

Partially addresses #14040. Pull request for Executive stakeholder stub will follow by 6/18/19.

### CODE REVIEW

File: docs/docs/winning-over-engineering-leaders.md [hunk 1]
```diff
@@ -1,15 +1,74 @@
...
+
+## Common concerns
+
+You may find that your engineering leaders and managers have some concerns about Gatsby. Here are some common examples and responses you can provide to help alleviate those concerns.
+
+### Support for open source software is often unreliable and relatively few projects offer long-term viability. How is Gatsby different?
+
+> Gatsby has an extremely active and communicative community with over 2,000 contributors and tens of thousands of developers building Gatsby sites. The process for filing issues and [contributing to Gatsby](/contributing/) is well-documented. [Gatsby's extensive documentation](/docs/) also includes various guides, tutorials, plugin and starter libraries, troubleshooting, and additional resources. Last but not least, the Gatsby open source project is supported by [Gatsby, Inc](https://www.gatsbyjs.com/). whose team is committed, full-time, to developing and maintaining Gatsby. Gatsby, Inc. also offers support services, training, and partnership opportunities. 
+
+### Our site uses a lot of dynamic content and components. Didn't you say Gatsby was a static site generator?
```
[the remark this answers is not in the document]
Comment (caitlinthefirst) [reply docs/docs/winning-over-engineering-leaders.md]: I agree with this and the previous comment that the "static site" references/explanation need some finessing. I'll spend a little time on it this evening. Thanks!

**Comment by caitlinthefirst** [comment]
Thanks for the edits, @gillkyle! They've been committed.

File: docs/docs/winning-over-engineering-leaders.md [hunk 2]
```diff
@@ -1,15 +1,74 @@
...
+
+- [Beyond Static: Haptic Media Uses Gatsby to Build a Dynamic Web App](/blog/2019-02-05-hapticmedia-case-study/)
+- [IBM Uses Gatsby to Manage Enterprise-Level Content](/blog/2018-12-17-ibm-case-study/#big-company-big-website)
+- [How we're migrating a government open data site to Gatsby](/blog/2019-02-08-government-open-data-site-with-gatsby/)
+
+For examples of Gatsby sites, [check out the Showcase](/showcase/).
+
+## Conclusion
+
+When you're trying to win over a stakeholder, your first instinct may be to focus completely on Gatsby's benefits. However, when you're speaking to engineering leaders and managers, it's important to be positive while still being honest about Gatsby's limitations. Setting expectations before they go out to do their own research will help them focus on finding solutions rather than faults. There is ample proof that Gatsby's pros outweigh the cons - don't be afraid to help your engineering leaders dig into that information. (And it doesn't hurt to get a few of their team members on your side. Check out the [Winning Over Developers](/docs/winning-over-developers/) page for more on that subject.)
```
Comment (marcysutton) [inline docs/docs/winning-over-engineering-leaders.md]: I wonder if we could reword this to encourage a slightly more positive conclusion. How about "There is ample proof that Gatsby works well for many websites - don't be afraid to help your engineering leaders dig into pros and cons."

**Comment by caitlinthefirst** [comment]
Lol! Well, I ended up just merging your suggestions. They looked good to me.

File: docs/docs/winning-over-engineering-leaders.md (hunk 2 above)
Comment (caitlinthefirst) [reply docs/docs/winning-over-engineering-leaders.md]: Made this and the previous edit. Thanks for the feedback!
